beta/kustomize/README.md

```diff
@@ -0,0 +1,214 @@
+# Deploy 1Password SCIM Bridge on Kubernetes with Kustomize
+
+_Learn how to deploy 1Password SCIM Bridge on a Kubernetes cluster using Kustomize._
+
+> **Note**
+>
+> If you use Azure Kubernetes Service, learn how to [deploy the SCIM bridge there](https://support.1password.com/scim-deploy-azure/).
+
+**Table of contents:**
+
+- [Before you begin](#before-you-begin)
+- [Step 1: Deploy 1Password SCIM Bridge to the Kubernetes cluster](#step-2-deploy-1password-scim-bridge-to-the-kubernetes-cluster)
+- [Step 2: Create a DNS record](#step-3-create-a-dns-record)
+- [Step 3: Configure Let's Encrypt](#step-4-configure-lets-encrypt)
+- [Step 4: Test the SCIM bridge](#step-5-test-the-scim-bridge)
+- [Step 5: Connect your identity provider](#step-6-connect-your-identity-provider)
+- [Update your SCIM Bridge](#update-your-scim-bridge)
+- [Appendix: Resource recommendations](#appendix-resource-recommendations)
+- [Appendix: Customize your deployment](#appendix-customize-your-deployment)
+
+## Before you begin
+
+Before you begin, familiarize yourself with [PREPARATION.md](/PREPARATION.md) and complete the necessary steps there.
+
+Additionally, as there are various subfolders needed, all paths will be based with a `.`.
+This `.` refers to the folder that contains this readme, and will be referred to as the Root Folder. For example, the Let's Encrypt
+options are referenced as `./overlays/letsencrypt`.
+
+### In this folder
+
+- [`/base`](./base/): The default deployment that the overlays build on top of to
+  deploy your preferred bridge.
+- [`/overlays`](./overlays/): The overlays to customize your deployment.
+- [`README.md`](./README.md): the document that you are reading. 👋😃
+
+## Step 1: Prepare your `scimsession` file
+
+So that Kustomize can create a secret from your `scimsession` file, place it in the
+`./base` directory. The full path should be `./base/scimsession`. Kubernetes
+will use this file along with a Kustomize secret generator to create the Kubernetes secret
+containing the contents of your `scimsession` file, named as `scimsession`.
+
+## Step 2: Determine which overlays to use
+
+There are currently 3 overlays defined for modifying the deployment of the bridge, in
+addition to the deployment overlay. In most standard deployments, the `letsencrypt`
+overlay is going to be used. This overlay configures the SCIM bridge to request a 
+certificate from Let's Encrypt using a domain name defined in the overlay ConfigMap. 
+If you're bringing your own TLS certificate, the `self-managed-tls` overlay is to be 
+used instead.
+
+While you will be enabling the Let's Encrypt overlay, there are final changes that will
+need to be made after the initial deployment.
+
+If you're using Google Workspace, you'll also need the `workspace` overlay configured
+as documented in that overlay's README.
+
+## Step 3: Configure the overlays
+
+### Self Managed TLS
+
+To bring your own TLS certificate for use with the SCIM bridge, add your TLS private
```

Comment:
Detail how this is still utilizing CertificateManager. 

You may supply your own TLS certificate with CertificateManager instead of invoking Let's Encrypt.